apps/riscv-tests/isa/Makefile

```diff
@@ -48,13 +57,13 @@ vpath %.S $(src_dir)
 	$(RISCV_OBJDUMP) $< > $@
 
 %.out: %
-	$(RISCV_SIM) --isa=rv64gcv --varch="vlen:2048,elen:64,slen:2048" $< 2> $@
+	$(RISCV_SIM) --isa=rv64gcv --varch="vlen:4096,elen:64,slen:4096" $< 2> $@
```

Comment:
Any specific reason behind this change? (Should we adapt it to Ara's?)

Reply:
It was to adapt it to Ara (always useful for debug purposes) :)